src/Cli/UpdateTerminusDocsCommands.php

```diff
@@ -144,14 +144,20 @@ public function terminusUpdateDocs($options = [
             file_put_contents($dir . '/source/data/terminusReleases.json', $releasesJson);
         }
 
+        $installMdContents = file_get_contents($dir . '/source/content/terminus/install.md');
+        $replacement = '${1}' . $terminusRelease . '${3}';
+        $installMdContents = preg_replace('/(.*)(\d\.\d\.\d)(.*)/', $replacement, $installMdContents);
```

Comment:
This is a pretty long page; this regex could easily match the wrong part of the page if additional content were added in the future, e.g. a line somewhere higher on the page where a semver version of some other software is mentioned. Make this match a little more specific, i.e. include contexual text around the Terminus version, so that we're sure that we're replacing the right thing.

Reply:
Fixed, thanks